src/reference-designs/tko-on-vsphere-with-tanzu-nsxt.md

```diff
@@ -0,0 +1,446 @@
+   <td>Istio metrics including request rates, trace rates, throughput, etc.</td>
+   <td>Request rate (transactions per second)</td>
+  </tr>
+</table>
+
+Some sample dashboards are shown below. Tanzu Observability can display metric data from the full stack of application elements from the platform (VMware ESXi servers), to the virtual environment, to the application environment (Kubernetes), and down to the various components of an application (APM).
+
+#### ESXi Dashboards
+
+![ESXi Host summary](img/tko-on-vsphere-with-tanzu-nsxt/image15.png)
+
+![ESXi CPU usage](img/tko-on-vsphere-with-tanzu-nsxt/image16.png)
+
+![ESXi Memory usage](img/tko-on-vsphere-with-tanzu-nsxt/image17.png)
+
+![ESXi Storage performance](img/tko-on-vsphere-with-tanzu-nsxt/image18.png)
+
+![ESXi Network performance](img/tko-on-vsphere-with-tanzu-nsxt/image19.png)
+
+#### VM Dashboards
+
+![VM CPU usage](img/tko-on-vsphere-with-tanzu-nsxt/image20.png)
+
+![VM Memory usage](img/tko-on-vsphere-with-tanzu-nsxt/image21.png)
+
+![VM Disk performance](img/tko-on-vsphere-with-tanzu-nsxt/image22.png)
+
+![VM Network performance](img/tko-on-vsphere-with-tanzu-nsxt/image23.png)
+
+#### Storage Dashboards
+
+![Datastore performance](img/tko-on-vsphere-with-tanzu-nsxt/image24.png)
+
+#### Kubernetes Dashboards
+
+![Summary for one or more Kubernetes clusters](img/tko-on-vsphere-with-tanzu-nsxt/image25.png)
+
+![Cluster level metrics of a specific cluster](img/tko-on-vsphere-with-tanzu-nsxt/image26.png)
+
+#### Application Dashboards
+
+![Istio Service Mesh summary (example)](img/tko-on-vsphere-with-tanzu-nsxt/image27.png)
+
+![Istio Service Mesh details (example)](img/tko-on-vsphere-with-tanzu-nsxt/image28.png)
+
+![Application Overview - microservices aware out of the box.](img/tko-on-vsphere-with-tanzu-nsxt/image29.png)
+
+![Detailed Application Dashboards - out of the box.](img/tko-on-vsphere-with-tanzu-nsxt/image30.png)
+
+![Automatic Distributed Tracing information between (micro)services.](img/tko-on-vsphere-with-tanzu-nsxt/image31.png)
+
+To view and download integrations with prebuilt dashboards that are available in Wavefront, see [Integrations](https://vmware.wavefront.com/integrations) on the VMware Tanzu Observability site.
+
+## Tanzu Service Mesh (TSM)
+
+VMware Tanzu Service Mesh is an enterprise-class service mesh solution that provides consistent control and security for microservices, end-users, and data across all your clusters and clouds in the most demanding multi-cluster and multi-cloud environments.
+
+### Key Benefits of Tanzu Service Mesh:
+
+- Extends the service mesh capability (discovery, connectivity, control, security, and observability) to users and data.
+- Facilitates the development and management of distributed applications across multiple clusters, multiple clouds, and in hybrid-cloud environments with Global Namespaces, supporting federation across organizational boundaries, technologies, and service meshes.
```

Comment:
Change Global Namespaces to global namespaces.

Reply:
done